src/pages/search.tsx

```diff
@@ -60,7 +60,7 @@ const Search: NextPage<SearchProps> = ({ languages, translations }) => {
     () => ({
       page: currentPage,
       languages: selectedLanguages,
-      query: debouncedSearchQuery,
+      q: debouncedSearchQuery,
```

Comment:
quick question, wondering if we should consider the backward compatibility?

previous version is using the `query` search param I think ? https://previous.quran.com/search?query=noah